frontend/class-monthly-report.php

```diff
@@ -220,7 +220,7 @@
                     var year = $('#year option:selected').text();
                     var grade = $('#grade option:selected').text();
                     var selectedClass = $('#classDropdown option:selected').text();
-                    month = "0" + (parseInt(month) + 1);
+                    month = parseInt(month)<9 ? "0" + (parseInt(month) + 1) : (parseInt(month) + 1)
```

Comment:
Should be <= right?
And there is another file to fix mentioned in the issue

Reply:
If you put the equal sign, then it doesn't works for the 10th month, and will add the zero to it, have you tested this fix in your code?

P. S. The other file you mentioned is fixed with this solution too